foyer/forcefields/xml/oplsaa.xml

```diff
@@ -464,11 +464,11 @@
   <Type name="opls_461" class="opls_461" element="C" mass="12.011"/>
   <Type name="opls_462" class="opls_462" element="H" mass="1.008"/>
   <Type name="opls_463" class="opls_463" element="C" mass="12.011"/>
-  <Type name="opls_465" class="opls_465" element="C" mass="12.011"/>
-  <Type name="opls_466" class="opls_466" element="O" mass="15.9994"/>
-  <Type name="opls_467" class="opls_467" element="O" mass="15.9994"/>
-  <Type name="opls_468" class="opls_468" element="C" mass="12.011"/>
-  <Type name="opls_469" class="opls_469" element="H" mass="1.008"/>
+  <Type name="opls_465" class="C_2" element="C" mass="12.011" def="[C;X3](O)(O)" desc="C in C=O in ester" doi="10.1021/ja9539195"/>
```

Comment:
It seems like this atomtype might be a bit too general. It seems to also match for a few carboxylic acids. In this case formic acid and propanoic acid.

Reply:
we probably need an override for the carboxylic case